whipper/test/test_common_mbngs.py

```diff
@@ -44,6 +44,25 @@ def testTrackTitle(self):
         track1 = metadata.tracks[0]
         self.assertEqual(track1.title, 'Brownsville Turnaround')
 
+    def testClassicalTrack(self):
+        """
+        Test if composers and performers are extracted properly
+        see https://github.com/whipper-team/whipper/issues/191
+        """
+        # Using: Début Recital - Martha Argerich
+        # https://musicbrainz.org/release/5598dd88-5a14-4661-a9cc-fe37a5246028
+        filename = 'whipper.release.5598dd88-5a14-4661-a9cc-fe37a5246028.json'
+        path = os.path.join(os.path.dirname(__file__), filename)
+        with open(path, "rb") as handle:
+            response = json.loads(handle.read().decode('utf-8'))
+
+        metadata = mbngs._getMetadata(response['release'], classical=True)
+        track1 = metadata.tracks[0]
+        self.assertEqual(track1.composers, ['Fryderyk Chopin'])
+        self.assertEqual(track1.performers, ['balance: Heinz Wildhagen',
+                                             'piano: Martha Argerich',
+                                             'producer: Karl-Heinz Schneider'])
```

Comment:
… How does "producer" end up as a performance relationship?

Reply:
Damn, my bad :<

Reply:
This should be fixed now 🗡